src/wasm-linker.h

```diff
@@ -173,6 +178,9 @@ class LinkerObject {
 
   std::map<Name, Address> segments; // name => segment index (in wasm module)
 
+  // preassigned indexes for functions called indirectly
+  std::map<Name, uint64_t> indirectIndexes;
```

Comment:
I think this can have `Address` type instead of `uint64_t` like other address-sized things in wasm.